OpenRA.Mods.Common/Activities/Attack.cs

```diff
@@ -114,7 +114,7 @@ protected virtual Activity InnerTick(Actor self, AttackBase attack)
 				return NextActivity;
 			}
 
-			var targetedPosition = attack.Info.AttackTargetCenter ? Target.CenterPosition : Target.Positions.PositionClosestTo(pos);
+			var targetedPosition = attack.GetTargetPosition(pos, Target, true);
```

Comment:
I've looked at it but as far as I can tell, I'd just have to do the same thing GetTargetPosition does, so that would only result in more lines/duplication.